### DISCUSSION TITLE
Chrome 118

### DISCUSSION THREAD

**Discussion by krausest** [krausest/js-framework-benchmark#1427] [Announcements]
Chrome 118 [results](https://krausest.github.io/js-framework-benchmark/2023/table_chrome_118.0.5993.70.html) are published. 

Please note that there have been significant changes for this run:
* We're now using a [weighted geometric](https://github.com/krausest/js-framework-benchmark/wiki/Computation-of-the-weighted-geometric-mean) mean for the overall result
* The number of runs for the CPU benchmarks was bumped from 10 to 15 for all benchmarks except select row, which was bumped to 25 runs. We hope this improves accuracy of the benchmarks

So please don't hesitate to post in the discussion sections if you find something suspicious.

Attached you find a binary build of all frameworks such that you can run the benchmark without installing and building all frameworks (see README for how to use the build.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/krausest/js-framework-benchmark/releases/tag/chrome118'>Chrome 118</a>.</em>

**Comment by maomincoding** [comment]
Is this the preliminary result？

**Reply by maomincoding** [reply]
[https://krausest.github.io/js-framework-benchmark/index.html](https://krausest.github.io/js-framework-benchmark/index.html)
The latest official results are not displayed on this page.

**Reply by krausest** [reply]
Thanks for the heads up. I'll fix both this evening.

**Reply by maomincoding** [reply]
Thanks for your hard work！

**Reply by krausest** [reply]
Results table and index.html were updated. The only change in the table is that it's (again) marked as an official result.

**Reply by krausest** [reply]
P.S: If you can't see it please force a reload in the browser.

**Reply by maomincoding** [reply]
Thanks !
